docs/user-guide/deployment-scenarios.md

```diff
@@ -22,15 +22,105 @@ updates](station-firmware-upgrade.md) for example.
 
 ![Keep data on-premise](../images/scenarios/scenario-local-data.png)
 
-## Redundancy with multiple concentrators and gateways
+## Scalability & Availability
 
-In a setup where sensors can reach at least one concentrator, it is possible
-to get redundancy by using at least two concentrators per gateway, and at least
-two gateways. In this case, the deduplication of the messages will happen in two
-levels: the Function will signal if a message is a duplicate, and the gateway
-itself will handle the deduplication. It is possible to configure how
-deduplication is handled and which strategy is used.  
-([More info on deduplication strategies](../adr/007_message_deduplication.md))
+### Single conentrator with single LNS
+
+This is the most basic deployment you can chose. It does only deploy 1 concentrator
+and one network server. Each sensor can at most reach a single concentrator.
+
+#### Advantages
+
+1. Scalability: this model scales very well. There is no additional overhead to deduplicate
+messages, nor do we need to determine who is winning the race to potentially send messages
+back to the device. Also, no additional edge hub keeps a connection for the leaf device, which helps with the scale requirement on the IoT Hub itself.
+1. Simple: the deployment is simple to start with and maintain.
+1. Low cost: No redundant hardware is used, which helps keeping the operational cost down.
+
+#### Disadvantages
+
+1. Availability: this solution has 2 points of failure: the concentrator and the
```

Comment:
Consider rephrasing it to two additional points of failure, since there may also be other points of failure in our system (Function, IoT Hub, Redis, etc).

Reply:
added a general note on the other services. This is only focusing on LBS/LNS. Other services have dedicated documentation on HA and scalability, that should be taken into considation.

c687a555748d05d84a33a4555a06c878e2cc1df8